### ISSUE TITLE
Make the tutorial more clear

### ISSUE THREAD

**Issue by giorgian** [habitat-sh/habitat#1080]
There seems to be a missing step in the online tutorial that makes it hard to follow: 

<blockquote>
<ol start="5">
<li>Change the pkg_source value to point to the archive file that contains the source files described in the previous step.</li>
</ol>
</blockquote>

It would be nice to tell users to create the archive file before referencing it. For instance, should the archive include a folder or not? And, should I upload it on S3?

Later on point 6, it says:

> Note: If you modified the source files from the previous step and created your own archive, you will have to compute the sha256 value yourself.

This should mean that, if I didn't modify the source files, the sha256 value of my archive will be the one listed; will it? With or without folder? Am I using the same umask?

**Comment by metadave** [comment]
ping @davidwrede

**Comment by davidwrede** [comment]
Sorry you're running into problems with the tutorial, @giorgian. You don't need to create an archive file and upload it if you plan on sticking with the default node.js app that's described in the tutorial. The source files are already in an S3 bucket for people to download as they build their tutorial plan. 

And to your other question, yes, if you didn't modify anything and create your own archive, then the checksum value listed is the one you should use.

I'll add some clarification to the docs this week.

**Comment by smurawski** [comment]
The tutorial has changed to working with a cloned git repo, so this shouldn't be an issue anymore.

Thanks for catching it!
